### ISSUE TITLE
Recover mobile screen space

### ISSUE THREAD

**Issue by martinhannigan** [peeringdb/peeringdb#1571]
Drop advertising from mobile screen. Takes up too much screen space and makes mobile experience less useful.

![image](https://github.com/peeringdb/peeringdb/assets/21081154/51a66bc2-d7fc-4ff1-8cc8-270adbf715ad)

**Comment by leovegoda** [comment]
For all users or anonymous users?

**Comment by martinhannigan** [comment]
I personally would prefer no-one sees an ad.  Even ours.

I wouldn't object to registered users not seeing it.

Tks

-M<

**Comment by job** [comment]
I think indeed it makes sense to only show this description to users who aren’t logged in, as there is the fair chance such visitors aren’t familiar with peeringdb.

Insightful suggestion!

**Comment by leovegoda** [comment]
The current design for the updated web UX has logged in users seeing search and the current numbers for each kind of object. Only anonymous users see the blurb describing PeeringDB. We can adjust the design between user testing and deployment based on feedback

**Comment by martinhannigan** [comment]
Hi Leo-Does this mean today, everyone sees the ad as described? And next UI
update will serve it to only “anonymous” users ?

Can you also clarify anonymous? Do you simply mean non logged in users?
Based on the way the Internet and PDB works I’m surprised if viewers of PDB
are anonymous unless they take deliberate steps. We keep logs and we use
analytics. Can you clarify that?


Thanks!

-M<

**Comment by leovegoda** [comment]
Yes. Everyone sees the about blurb today. The current plan is that only anonymous users would see it in the new web UX. And anonymous means not logged in, nothing more than that.
